Assets/Scripts/ManaBarUI.cs

```diff
@@ -1,20 +1,29 @@
-﻿using System.Collections;
+﻿using System;
+using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 
```

Comment:
i added [EasyButtons](https://github.com/madsbangh/EasyButtons) to this project to make it easier to test code. If you include `using EasyButtons` and add the attribute `[Button]` right above each function, you'll be able to run them by clicking a button in the inspector. You can also limit it so that the buttons can't be clicked during edit mode with `[Button(mode = buttonmodes.EnabledInPlayMode)]` or something close to that i forgot let me know if you need help